portal-web/test/functional/com/liferay/portalweb/tests/enduser/headless/HeadlessDiscovery/HeadlessDiscovery.testcase

```diff
@@ -369,10 +367,7 @@ definition {
 			var siteId = JSONGroupAPI._getGroupIdByNameNoSelenium(
 				groupName = "Guest",
 				site = "true");
-			var requestBody = '''{
-                "contentStructureId": ${ddmStructureId},
-                "title": "title of structured content"
-            }''';
+			var requestBody = '''{"contentStructureId": ${ddmStructureId},"title": "title of structured content"}''';
```

Comment:
aligning it is a nice idea, but I think we lose readability this way, is it possible to make it one `key:value` per line?